app/views/compute_resources_vms/form/vmware/_scsi_controller.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= select_f f, :type, compute_resource.scsi_controller_types, :first, :last, {},
+  { :label => _("Controller type"),:class => "col-md-2", :disabled => !new_host } %>
+  
+<%= f.hidden_field :key, :value => f.object.key %>
+דדד
```

Comment:
unknown extra chars?

do we need this partial at all? I thought this is done via react now?